eo-maven-plugin/src/main/java/org/eolang/maven/optimization/Optimization.java

```diff
@@ -24,13 +24,14 @@
 package org.eolang.maven.optimization;
 
 import com.jcabi.xml.XML;
-import java.util.function.Function;
+import java.nio.file.Path;
+import org.cactoos.Func;
 
 /**
  * Abstraction for XML optimizations.
  *
  * @since 0.28.11
  */
 @FunctionalInterface
-public interface Optimization extends Function<XML, XML> {
```

Comment:
@Yanich96 I don't understand why we have to change this to `Path`. Can we just save `lastModifiedTime` to `XMIR` and then just read it?